API/Features/Image/Requests/PutOrPatchImage.cs

```diff
@@ -0,0 +1,378 @@
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using DLCS.Core.Settings;
+using DLCS.Core.Strings;
+using DLCS.Model.Assets;
+using DLCS.Model.Messaging;
+using DLCS.Model.Spaces;
+using DLCS.Model.Storage;
+using MediatR;
+using Microsoft.Extensions.Options;
+
+namespace API.Features.Image.Requests
+{
+    /// <summary>
+    /// Unlike Deliverator we will handle PUTs and PATCHes with a single command, with
```

Comment:
If you're unfamiliar with Deliverator this doesn't make sense - feels more like a remark, or comment than summary.